datahub-web-react/src/app/entity/shared/containers/profile/sidebar/SidebarAboutSection.tsx

```diff
@@ -117,6 +124,23 @@ export const SidebarAboutSection = ({ properties }: { properties?: Props }) => {
                     )}
                 </SidebarLinkList>
             )}
+            {sourceRef && (
```

Comment:
Should we check that sourceUrl exists also?

Reply:
(Or what will happen?)

Reply:
in theory it could happen, if they only include `sourceRef` in the yaml and not `sourceUrl` - however I do have a check that if there isn't a `sourceUrl` then we just display the ref name with no link (I checked and this is how it used to work on an old component `GlossarySidebarAboutSection`)

Reply:
Okay got it cool